Makefile

```diff
@@ -76,11 +92,18 @@ prettier: npm
 
 all: hub sensor
 
+$(SENSOR_OBJ_DIR)/%.o: $(SENSOR_SRC_DIR)/%.cpp
```

Comment:
I don't think we need this target or the next one, they were removed a few commits ago. Was there a reason to add them back in?

Reply:
I didnt add this one for sure. Yo can take this one out.